### PR TITLE
perf report for 2023-09-13

### PR DESCRIPTION
**Pull request by pnkfelix**

### CODE REVIEW

File: draft/2023-09-13-this-week-in-rust.md
Comment (mariannegoldin) [inline draft/2023-09-13-this-week-in-rust.md:89]: This link seems to be broken - is there another target we should point to?

**Review by mariannegoldin — COMMENTED** [review]
Thank you for this submission. As soon as we can update the full report link target, I will approve and merge.

**Comment by pnkfelix** [comment]
> Thank you for this submission. As soon as we can update the full report link target, I will approve and merge.

The full report will be available once https://github.com/rust-lang/rustc-perf/pull/1719 is merged. I’ll go check if there’s someone who can approve it (though I think there may be precedent to allow this notice to be merged to TWIR before the rustc-perf PR is merged…)

**Comment by mariannegoldin** [comment]
@pnkfelix thanks for the update and the PR link! We might post the summary, and then once the PR goes through, add the link to the full report.

File: draft/2023-09-13-this-week-in-rust.md
Comment (mariannegoldin) [inline draft/2023-09-13-this-week-in-rust.md:89]:
```suggestion
[Full report  - pending](https://github.com/rust-lang/rustc-perf/blob/master/triage/2023-09-13.md)
```

**Review by mariannegoldin — APPROVED** [review]
Approving and merging with "pending" annotation for link - to be updated once the report is published.

**Pull request merged**
